tests/xharness/Jenkins/TestTasks/AggregatedRunSimulatorTask.cs

```diff
@@ -94,7 +96,7 @@ protected override async Task ExecuteAsync ()
 				if (device != null)
```

Comment:
```suggestion
				if (device is not null)
```
Might as well throw these in there :)